lib/fluent/plugin/filter_throttle.rb

```diff
@@ -0,0 +1,137 @@
+require 'fluent/filter'
+
+module Fluent
+  class ThrottleFilter < Filter
+    Fluent::Plugin.register_filter('throttle', self)
+
+    config_param :group_key, :string, :default => 'kubernetes.container_name'
+    config_param :group_bucket_period_s, :integer, :default => 60
+    config_param :group_bucket_limit, :integer, :default => 6000
```

Comment:
Could we please add documentation to all the parameters?

Reply:
PR description: Refer to [README.md](/rubrikinc/fluent-plugin-throttle/blob/master/README.md) for some extra documentation.